career-mentorship-program/2021-spring/matching_code.py

```diff
@@ -175,6 +194,21 @@ class Mentee(Student):
     # Which areas of the recruitment process the mentee wants help with
     desired_topics: Set[str]
 
+    # Hours per week the mentor is able to dedicate
```

Comment:
This comment needs to be changed to reflect that it's not hours per week, but actually a scale.